libafl/src/executors/inprocess.rs

```diff
@@ -1153,17 +1153,17 @@ mod windows_exception_handler {
             compiler_fence(Ordering::SeqCst);
         }
 
-        let code = ExceptionCode::try_from(
-            exception_pointers
-                .as_mut()
-                .unwrap()
-                .ExceptionRecord
-                .as_mut()
-                .unwrap()
-                .ExceptionCode
-                .0,
-        )
-        .unwrap();
+        let code =if let Some(exception_pointers) = exception_pointers.as_mut() {
+            ExceptionCode::try_from(
+                exception_pointers
+                    .ExceptionRecord
+                    .as_mut()
+                    .unwrap()
+                    .ExceptionCode
+                    .0).unwrap()
+        } else {
+            ExceptionCode::NotImplemented // Should be changed probably
+        };
 
         #[cfg(feature = "std")]
         eprintln!("Crashed with {}", code);
```

Comment:
this var `code` is only used here. So I think we could probably do something like

if let Some(c) = exception_pointer.code.0 {
 eprintln!("Crashed with {}", c);
}
else{
eprintln!("Crashed with SIGABRT");
}